source/simulator/assembly.cc

```diff
@@ -1286,6 +1286,20 @@ namespace aspect
           return residuals;
         }
 
+        double
+        dg_approximation_to_face_measure(const typename DoFHandler<2>::face_iterator face) const
+        {
```

Comment:
can you add a comment before the first of the two functions that explains where this is from? (Namely, deal.II's TriaAccessor::measure, which implements the Q1 mapping case.)